fdbclient/FDBTypes.h

```diff
@@ -745,15 +772,17 @@ struct TLogVersion {
 		// V3 was the introduction of spill by reference;
 		// V4 changed how data gets written to satellite TLogs so that we can peek from them;
 		// V5 merged reference and value spilling
+		// V6 updated serialization format of mutations sent from proxy to tlogs with added transaction info
 		// V1 = 1,  // 4.6 is dispatched to via 6.0
 		V2 = 2, // 6.0
 		V3 = 3, // 6.1
 		V4 = 4, // 6.2
 		V5 = 5, // 6.3
+		V6 = 6, // 7.0
 		MIN_SUPPORTED = V2,
-		MAX_SUPPORTED = V5,
-		MIN_RECRUITABLE = V4,
-		DEFAULT = V5,
+		MAX_SUPPORTED = V6,
+		MIN_RECRUITABLE = V5,
+		DEFAULT = V6,
```

Comment:
```suggestion
		DEFAULT = V5,
```

If you immediately default to the new version, you'll break rollback.

Reply:
This stack is likely going to be closed in favor of #3700.

Reply:
Made this change in #3748. Closing this PR.